app/scripts/components/dashboard/charts/metric-status.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import T from '../../utils/t';
+
+require('stylesheets/dashboard/charts/metric-status');
+
+const MetricStatus = React.createClass({
+  propTypes: {
+    metric: React.PropTypes.number.isRequired,
+    title: React.PropTypes.string.isRequired,
+    total: React.PropTypes.number.isRequired,
+  },
+
+  render() {
+    const classes = classNames({
+      'icon': true,
+      'green': false, // some conditions
```

Comment:
Do we have info yet for how to rank these?

I guess we should do another rank service like in education, but that would be a separate feature.